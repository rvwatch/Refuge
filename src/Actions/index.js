export const addUser = user => ({
  type: 'ADD_USER',
  user
});
