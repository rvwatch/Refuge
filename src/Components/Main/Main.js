import React from 'react';
import { Link, withRouter } from 'react-router-dom';
import { connect } from 'react-redux';
import Charts from '../../Containers/Charts/Charts';

export const Main = (props) => {
  const charts =
  props.heartRate.length && props.stepsTaken.length ? (
    <Charts />
  ) : (
    'Loading'
  );
   return (
    <div>
      {charts}
      <section className="therapies-wrap">
        <h2>therapies:</h2>
        <Link to="/breathing" className="breath">
          <h3>breathing exercises</h3>
          <img
            alt="Icon"
            className="lungs-icon"
            src={require('../../Assets/images/lungs.svg')}
          />
        </Link>
        <Link to="/mindfulness" className="mind">
          <h3>guided meditations</h3>
          <img
            alt="Icon"
            className="mindfulness-icon"
            src={require('../../Assets/images/meditation.svg')}
          />
        </Link>
        <Link to="/sound" className="sound">
          <h3>calming playlists</h3>
          <img
            alt="Icon"
            className="sound-icon"
            src={require('../../Assets/images/headphones.svg')}
          />
        </Link>
        <Link to="/journal" className="notes">
          <h3>journal</h3>
          <img
            alt="Icon"
            className="notes-icon"
            src={require('../../Assets/images/book.svg')}
          />
        </Link>
        <Link to="/videos" className="sight">
          <h3>relaxing videos</h3>
          <img
            alt="Icon"
            className="sight-icon"
            src={require('../../Assets/images/eyeglasses.svg')}
          />
        </Link>
        <Link to="/support" className="lifeline">
          <h3>support</h3>
          <img
            alt="Icon"
            className="phone-icon"
            src={require('../../Assets/images/call-answer.svg')}
          />
        </Link>
      </section>
    </div>
  );
};

export const mapStateToProps = (state, ownProps) => {
  return {
    heartRate: state.heartRate,
    stepsTaken: state.stepsTaken
  }
};

export default withRouter(connect(mapStateToProps, null)(Main));
