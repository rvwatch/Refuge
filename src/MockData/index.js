export const mockHeartRate = [
  { time: '00:00:00', value: 57 },
  { time: '00:01:00', value: 59 },
  { time: '00:02:00', value: 58 },
  { time: '00:03:00', value: 59 },
  { time: '00:04:00', value: 59 },
  { time: '00:05:00', value: 59 }
];

export const mockChartData = [
  { x: '12:00:00 A.M.', y: 57 },
  { x: '12:01:00 A.M.', y: 59 },
  { x: '12:02:00 A.M.', y: 58 },
  { x: '12:03:00 A.M.', y: 59 },
  { x: '12:04:00 A.M.', y: 59 },
  { x: '12:05:00 A.M.', y: 59 }
];

export const mockStepsTaken = [
  { time: '00:00:00', value: 0 },
  { time: '00:01:00', value: 0 },
  { time: '00:02:00', value: 0 },
  { time: '00:03:00', value: 0 },
  { time: '00:04:00', value: 0 },
  { time: '00:05:00', value: 0 },
  { time: '00:06:00', value: 0 },
  { time: '00:07:00', value: 0 }
];

export const mockFitbitProfile = {
  user: 'Ricardo V.',
  avgSteps: 8150
};

export const restingHeartRate = '72.34';

export const user = { username: 'ricardo' };
