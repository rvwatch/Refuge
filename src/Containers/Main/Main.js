import React from 'react';
import { Link, withRouter } from 'react-router-dom';
import { connect } from 'react-redux';
import Charts from '../Charts/Charts';
import { array, bool } from 'prop-types';
import * as Actions from '../../Actions/index';

export const Main = (props) => {
  
  const charts =
  props.heartRate.length && props.stepsTaken.length ?
    <Charts /> : <h2 className='loading'>Loading Charts</h2>;
  
  return (
    <section className='main-wrap'>
      {charts}
      <section className="therapies-wrap">
        <h2>therapies:</h2>
        <Link to="/breathing" className="breath">
          <h3>breathing exercises</h3>
          <img
            alt="Icon"
            className="lungs-icon"
            src={require('../../Assets/images/lungs.svg')}
          />
        </Link>
        <Link to="/mindfulness" className="mind">
          <h3>guided meditations</h3>
          <img
            alt="Icon"
            className="mindfulness-icon"
            src={require('../../Assets/images/meditation.svg')}
          />
        </Link>
        <Link to="/sound" className="sound">
          <h3>calming playlists</h3>
          <img
            alt="Icon"
            className="sound-icon"
            src={require('../../Assets/images/headphones.svg')}
          />
        </Link>
        <Link to="/journal" className="notes">
          <h3>journal</h3>
          <img
            alt="Icon"
            className="notes-icon"
            src={require('../../Assets/images/book.svg')}
          />
        </Link>
        <Link to="/videos" className="sight">
          <h3>relaxing videos</h3>
          <img
            alt="Icon"
            className="sight-icon"
            src={require('../../Assets/images/eyeglasses.svg')}
          />
        </Link>
        <a className="lifeline" href="tel:555-555-5555">
          <h3>support</h3>
          <img
            alt="Icon"
            className="phone-icon"
            src={require('../../Assets/images/call-answer.svg')}
          />
        </a>
        
      </section>
    </section>
  );
};

export const mapStateToProps = 
({heartRate, stepsTaken, loggedIn}) => ({heartRate, stepsTaken, loggedIn});


export const mapDispatchToProps = dispatch => ({
  addFitBitData: data => dispatch(Actions.addFitBitData(data)),
  addHeartRate: heartRate => dispatch(Actions.addHeartRate(heartRate)),
  addStepsTaken: stepsTaken => dispatch(Actions.addStepsTaken(stepsTaken)),
  addRestingHeart: restingHeart =>
    dispatch(Actions.addRestingHeart(restingHeart)),
  logoutUser: () => dispatch(Actions.logoutUser())
});

Main.propTypes = {
  heartRate: array,
  stepsTaken: array,
  loggedIn: bool
};

export default withRouter(connect(mapStateToProps, mapDispatchToProps)(Main));
