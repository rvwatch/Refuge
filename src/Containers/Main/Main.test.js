import React from 'react';
import { Main, mapStateToProps, mapDispatchToProps } from './Main';
import { shallow } from 'enzyme';
import * as Actions from '../../Actions/';
import * as mock from '../../MockData/';

describe('Main', () => {
  let heartRate;
  let stepsTaken;
  let wrapper;
  beforeEach(() => {
    heartRate = mock.mockHeartRate;
    stepsTaken = mock.mockStepsTaken;
    wrapper = shallow(<Main heartRate={heartRate} stepsTaken={stepsTaken} />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should match the snapshot', () => {
    const heartRate = [];
    const stepsTaken = [];
    wrapper = shallow(<Main heartRate={heartRate} stepsTaken={stepsTaken} />);
    expect(wrapper).toMatchSnapshot();
  });

  describe('mapStateToProps', () => {
    it('should add heartRate data to props', () => {
      const expected = mock.mockHeartRate;
      const mockState = { heartRate: mock.mockHeartRate };
      const mapped = mapStateToProps(mockState);
      expect(mapped.heartRate).toEqual(expected);
    });

    it('should add stepsTaken data to props', () => {
      const expected = mock.mockStepsTaken;
      const mockState = { stepsTaken: mock.mockStepsTaken };
      const mapped = mapStateToProps(mockState);
      expect(mapped.stepsTaken).toEqual(expected);
    });
  });

  describe('mapDispatchToProps', () => {
    it('should call dispatch with correct params in addFitBitData', () => {
      const mockDispatch = jest.fn();
      const mockfitbitData = mock.mockFitbitProfile;
      const expected = Actions.addFitBitData(mockfitbitData);
      const mapped = mapDispatchToProps(mockDispatch);
      mapped.addFitBitData(mockfitbitData);
      expect(mockDispatch).toHaveBeenCalledWith(expected);
    });

    it('should call dispatch with correct params in addHeartRate', () => {
      const mockDispatch = jest.fn();
      const mockHeartRate = mock.mockHeartRate;
      const expected = Actions.addHeartRate(mockHeartRate);
      const mapped = mapDispatchToProps(mockDispatch);
      mapped.addHeartRate(mockHeartRate);
      expect(mockDispatch).toHaveBeenCalledWith(expected);
    });

    it('should call dispatch with correct params in addStepsTaken', () => {
      const mockDispatch = jest.fn();
      const mockStepsTaken = mock.mockStepsTaken;
      const expected = Actions.addStepsTaken(mockStepsTaken);
      const mapped = mapDispatchToProps(mockDispatch);
      mapped.addStepsTaken(mockStepsTaken);
      expect(mockDispatch).toHaveBeenCalledWith(expected);
    });

    it('should call dispatch with correct params in addRestingHeart', () => {
      const mockDispatch = jest.fn();
      const restingHeartRate = mock.restingHeartRate;
      const expected = Actions.addRestingHeart(restingHeartRate);
      const mapped = mapDispatchToProps(mockDispatch);
      mapped.addRestingHeart(restingHeartRate);
      expect(mockDispatch).toHaveBeenCalledWith(expected);
    });

    it('should call dispatch for logoutUser action', () => {
      const mockDispatch = jest.fn();
      const expected = Actions.logoutUser();
      const mapped = mapDispatchToProps(mockDispatch);
      mapped.logoutUser();
      expect(mockDispatch).toHaveBeenCalledWith(expected);
    });
  });
});
