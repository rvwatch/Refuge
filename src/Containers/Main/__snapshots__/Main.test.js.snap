// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    heartRate={
        Array [
            Object {
              "time": "00:00:00",
              "value": 57,
            },
            Object {
              "time": "00:01:00",
              "value": 59,
            },
            Object {
              "time": "00:02:00",
              "value": 58,
            },
            Object {
              "time": "00:03:00",
              "value": 59,
            },
            Object {
              "time": "00:04:00",
              "value": 59,
            },
            Object {
              "time": "00:05:00",
              "value": 59,
            },
          ]
    }
    stepsTaken={
        Array [
            Object {
              "time": "00:00:00",
              "value": 0,
            },
            Object {
              "time": "00:01:00",
              "value": 0,
            },
            Object {
              "time": "00:02:00",
              "value": 0,
            },
            Object {
              "time": "00:03:00",
              "value": 0,
            },
            Object {
              "time": "00:04:00",
              "value": 0,
            },
            Object {
              "time": "00:05:00",
              "value": 0,
            },
            Object {
              "time": "00:06:00",
              "value": 0,
            },
            Object {
              "time": "00:07:00",
              "value": 0,
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(Component) />,
        <section
          className="therapies-wrap"
>
          <h2>
                    therapies:
          </h2>
          <Link
                    className="breath"
                    replace={false}
                    to="/breathing"
          >
                    <h3>
                              breathing exercises
                    </h3>
                    <img
                              alt="Icon"
                              className="lungs-icon"
                              src="lungs.svg"
                    />
          </Link>
          <Link
                    className="mind"
                    replace={false}
                    to="/mindfulness"
          >
                    <h3>
                              guided meditations
                    </h3>
                    <img
                              alt="Icon"
                              className="mindfulness-icon"
                              src="meditation.svg"
                    />
          </Link>
          <Link
                    className="sound"
                    replace={false}
                    to="/sound"
          >
                    <h3>
                              calming playlists
                    </h3>
                    <img
                              alt="Icon"
                              className="sound-icon"
                              src="headphones.svg"
                    />
          </Link>
          <Link
                    className="notes"
                    replace={false}
                    to="/journal"
          >
                    <h3>
                              journal
                    </h3>
                    <img
                              alt="Icon"
                              className="notes-icon"
                              src="book.svg"
                    />
          </Link>
          <Link
                    className="sight"
                    replace={false}
                    to="/videos"
          >
                    <h3>
                              relaxing videos
                    </h3>
                    <img
                              alt="Icon"
                              className="sight-icon"
                              src="eyeglasses.svg"
                    />
          </Link>
          <a
                    className="lifeline"
                    href="tel:555-555-5555"
          >
                    <h3>
                              support
                    </h3>
                    <img
                              alt="Icon"
                              className="phone-icon"
                              src="call-answer.svg"
                    />
          </a>
</section>,
      ],
      "className": "main-wrap",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              therapies:
</h2>,
            <Link
              className="breath"
              replace={false}
              to="/breathing"
>
              <h3>
                            breathing exercises
              </h3>
              <img
                            alt="Icon"
                            className="lungs-icon"
                            src="lungs.svg"
              />
</Link>,
            <Link
              className="mind"
              replace={false}
              to="/mindfulness"
>
              <h3>
                            guided meditations
              </h3>
              <img
                            alt="Icon"
                            className="mindfulness-icon"
                            src="meditation.svg"
              />
</Link>,
            <Link
              className="sound"
              replace={false}
              to="/sound"
>
              <h3>
                            calming playlists
              </h3>
              <img
                            alt="Icon"
                            className="sound-icon"
                            src="headphones.svg"
              />
</Link>,
            <Link
              className="notes"
              replace={false}
              to="/journal"
>
              <h3>
                            journal
              </h3>
              <img
                            alt="Icon"
                            className="notes-icon"
                            src="book.svg"
              />
</Link>,
            <Link
              className="sight"
              replace={false}
              to="/videos"
>
              <h3>
                            relaxing videos
              </h3>
              <img
                            alt="Icon"
                            className="sight-icon"
                            src="eyeglasses.svg"
              />
</Link>,
            <a
              className="lifeline"
              href="tel:555-555-5555"
>
              <h3>
                            support
              </h3>
              <img
                            alt="Icon"
                            className="phone-icon"
                            src="call-answer.svg"
              />
</a>,
          ],
          "className": "therapies-wrap",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "therapies:",
            },
            "ref": null,
            "rendered": "therapies:",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  breathing exercises
</h3>,
                <img
                  alt="Icon"
                  className="lungs-icon"
                  src="lungs.svg"
/>,
              ],
              "className": "breath",
              "replace": false,
              "to": "/breathing",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "breathing exercises",
                },
                "ref": null,
                "rendered": "breathing exercises",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Icon",
                  "className": "lungs-icon",
                  "src": "lungs.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  guided meditations
</h3>,
                <img
                  alt="Icon"
                  className="mindfulness-icon"
                  src="meditation.svg"
/>,
              ],
              "className": "mind",
              "replace": false,
              "to": "/mindfulness",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "guided meditations",
                },
                "ref": null,
                "rendered": "guided meditations",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Icon",
                  "className": "mindfulness-icon",
                  "src": "meditation.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  calming playlists
</h3>,
                <img
                  alt="Icon"
                  className="sound-icon"
                  src="headphones.svg"
/>,
              ],
              "className": "sound",
              "replace": false,
              "to": "/sound",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "calming playlists",
                },
                "ref": null,
                "rendered": "calming playlists",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Icon",
                  "className": "sound-icon",
                  "src": "headphones.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  journal
</h3>,
                <img
                  alt="Icon"
                  className="notes-icon"
                  src="book.svg"
/>,
              ],
              "className": "notes",
              "replace": false,
              "to": "/journal",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "journal",
                },
                "ref": null,
                "rendered": "journal",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Icon",
                  "className": "notes-icon",
                  "src": "book.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  relaxing videos
</h3>,
                <img
                  alt="Icon"
                  className="sight-icon"
                  src="eyeglasses.svg"
/>,
              ],
              "className": "sight",
              "replace": false,
              "to": "/videos",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "relaxing videos",
                },
                "ref": null,
                "rendered": "relaxing videos",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Icon",
                  "className": "sight-icon",
                  "src": "eyeglasses.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  support
</h3>,
                <img
                  alt="Icon"
                  className="phone-icon"
                  src="call-answer.svg"
/>,
              ],
              "className": "lifeline",
              "href": "tel:555-555-5555",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "support",
                },
                "ref": null,
                "rendered": "support",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Icon",
                  "className": "phone-icon",
                  "src": "call-answer.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "a",
          },
        ],
        "type": "section",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(Component) />,
          <section
            className="therapies-wrap"
>
            <h2>
                        therapies:
            </h2>
            <Link
                        className="breath"
                        replace={false}
                        to="/breathing"
            >
                        <h3>
                                    breathing exercises
                        </h3>
                        <img
                                    alt="Icon"
                                    className="lungs-icon"
                                    src="lungs.svg"
                        />
            </Link>
            <Link
                        className="mind"
                        replace={false}
                        to="/mindfulness"
            >
                        <h3>
                                    guided meditations
                        </h3>
                        <img
                                    alt="Icon"
                                    className="mindfulness-icon"
                                    src="meditation.svg"
                        />
            </Link>
            <Link
                        className="sound"
                        replace={false}
                        to="/sound"
            >
                        <h3>
                                    calming playlists
                        </h3>
                        <img
                                    alt="Icon"
                                    className="sound-icon"
                                    src="headphones.svg"
                        />
            </Link>
            <Link
                        className="notes"
                        replace={false}
                        to="/journal"
            >
                        <h3>
                                    journal
                        </h3>
                        <img
                                    alt="Icon"
                                    className="notes-icon"
                                    src="book.svg"
                        />
            </Link>
            <Link
                        className="sight"
                        replace={false}
                        to="/videos"
            >
                        <h3>
                                    relaxing videos
                        </h3>
                        <img
                                    alt="Icon"
                                    className="sight-icon"
                                    src="eyeglasses.svg"
                        />
            </Link>
            <a
                        className="lifeline"
                        href="tel:555-555-5555"
            >
                        <h3>
                                    support
                        </h3>
                        <img
                                    alt="Icon"
                                    className="phone-icon"
                                    src="call-answer.svg"
                        />
            </a>
</section>,
        ],
        "className": "main-wrap",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                therapies:
</h2>,
              <Link
                className="breath"
                replace={false}
                to="/breathing"
>
                <h3>
                                breathing exercises
                </h3>
                <img
                                alt="Icon"
                                className="lungs-icon"
                                src="lungs.svg"
                />
</Link>,
              <Link
                className="mind"
                replace={false}
                to="/mindfulness"
>
                <h3>
                                guided meditations
                </h3>
                <img
                                alt="Icon"
                                className="mindfulness-icon"
                                src="meditation.svg"
                />
</Link>,
              <Link
                className="sound"
                replace={false}
                to="/sound"
>
                <h3>
                                calming playlists
                </h3>
                <img
                                alt="Icon"
                                className="sound-icon"
                                src="headphones.svg"
                />
</Link>,
              <Link
                className="notes"
                replace={false}
                to="/journal"
>
                <h3>
                                journal
                </h3>
                <img
                                alt="Icon"
                                className="notes-icon"
                                src="book.svg"
                />
</Link>,
              <Link
                className="sight"
                replace={false}
                to="/videos"
>
                <h3>
                                relaxing videos
                </h3>
                <img
                                alt="Icon"
                                className="sight-icon"
                                src="eyeglasses.svg"
                />
</Link>,
              <a
                className="lifeline"
                href="tel:555-555-5555"
>
                <h3>
                                support
                </h3>
                <img
                                alt="Icon"
                                className="phone-icon"
                                src="call-answer.svg"
                />
</a>,
            ],
            "className": "therapies-wrap",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "therapies:",
              },
              "ref": null,
              "rendered": "therapies:",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    breathing exercises
</h3>,
                  <img
                    alt="Icon"
                    className="lungs-icon"
                    src="lungs.svg"
/>,
                ],
                "className": "breath",
                "replace": false,
                "to": "/breathing",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "breathing exercises",
                  },
                  "ref": null,
                  "rendered": "breathing exercises",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Icon",
                    "className": "lungs-icon",
                    "src": "lungs.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    guided meditations
</h3>,
                  <img
                    alt="Icon"
                    className="mindfulness-icon"
                    src="meditation.svg"
/>,
                ],
                "className": "mind",
                "replace": false,
                "to": "/mindfulness",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "guided meditations",
                  },
                  "ref": null,
                  "rendered": "guided meditations",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Icon",
                    "className": "mindfulness-icon",
                    "src": "meditation.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    calming playlists
</h3>,
                  <img
                    alt="Icon"
                    className="sound-icon"
                    src="headphones.svg"
/>,
                ],
                "className": "sound",
                "replace": false,
                "to": "/sound",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "calming playlists",
                  },
                  "ref": null,
                  "rendered": "calming playlists",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Icon",
                    "className": "sound-icon",
                    "src": "headphones.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    journal
</h3>,
                  <img
                    alt="Icon"
                    className="notes-icon"
                    src="book.svg"
/>,
                ],
                "className": "notes",
                "replace": false,
                "to": "/journal",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "journal",
                  },
                  "ref": null,
                  "rendered": "journal",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Icon",
                    "className": "notes-icon",
                    "src": "book.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    relaxing videos
</h3>,
                  <img
                    alt="Icon"
                    className="sight-icon"
                    src="eyeglasses.svg"
/>,
                ],
                "className": "sight",
                "replace": false,
                "to": "/videos",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "relaxing videos",
                  },
                  "ref": null,
                  "rendered": "relaxing videos",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Icon",
                    "className": "sight-icon",
                    "src": "eyeglasses.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3>
                    support
</h3>,
                  <img
                    alt="Icon"
                    className="phone-icon"
                    src="call-answer.svg"
/>,
                ],
                "className": "lifeline",
                "href": "tel:555-555-5555",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "support",
                  },
                  "ref": null,
                  "rendered": "support",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Icon",
                    "className": "phone-icon",
                    "src": "call-answer.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "a",
            },
          ],
          "type": "section",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
